### DISCUSSION TITLE
Motion of window movement

### DISCUSSION THREAD

**Discussion by shindgew** [rxhanson/Rectangle#959] [Ideas]
🤔 Maybe Nonlinear motion can make it look better while adjusting windows.

**Comment by rxhanson** [comment]
Some people like this kind of thing, some don't. I don't have the bandwidth to add this in, but I see no reason to turn down a good pull request, if it's configurable via Terminal command.

**Reply by shindgew** [reply]
Gonna make it beta with [animation](https://developer.apple.com/documentation/appkit/animation) maybe next year.
